frontend/src/messages/single-form/en-US.json

```
@@ -21,5 +21,11 @@
   "Player": {
     "globalRanking": "Global Ranking",
     "countryRanking": "Country Ranking"
+  },
+  "toast": {
+    "userError": "Fill in all required fields!",
```

```suggestion
    "userError": "Please fill in all required fields!",
```

Everything looks good, suggestion is optional but it delivers the tone lighter (which I would personally go for)